tests/manage/mcg/test_prefix_list.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.objectbucket import OBC
+from ocs_ci.ocs.bucket_utils import (
+    write_random_objects_in_pod,
+    copy_objects,
+    list_objects_from_bucket,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework.pytest_customization.marks import (
+    bugzilla,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@bugzilla("2068110")
```

Comment:
add skip based on version

Reply:
okay :+1:

Reply:
@parikshithb which version should be skipped? quite not sure when this feature was introduced

Reply:
okay got it